system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -73,6 +75,19 @@
                       (:concept-id coll1) (:concept-id coll2))]
              errors)))))
 
+;; Verify that granule indexing is not being changed too quickly.
```

Comment:
The key point is that we cannot change the search application to start using the new fields until all of the granules have been indexed. That means the developer needs to make the change to start indexing the new field in one sprint, and make the search application changes in the following sprint.

Can you modify the comment to make it clearer?